net/tcp/impl/posix.h

```diff
@@ -95,6 +95,12 @@ class SocketHandle : private SocketSystemInitializer {
     if (socket_ < 0) {
       BRICKS_THROW(SocketCreateException());  // LCOV_EXCL_LINE -- Not covered by unit tests.
     }
+#ifdef BRICKS_APPLE
```

Comment:
This is a critical change, but we now have some inconsistency as for whether this logic should be in the `SocketHandle` or `Socket` constructor. The rest is all in `Socket` (search for `REUSEADDR`, for example). Perhaps we could keep everything in one place?